pkg/net/libp2p/key.go

```diff
@@ -27,13 +27,12 @@ func operatorPrivateKeyToNetworkKeyPair(operatorPrivateKey *operator.PrivateKey)
 		return nil, nil, fmt.Errorf("libp2p supports only secp256k1 operator keys")
 	}
 
-	// Libp2p keys are actually btcec keys under the hood.
-	btcecPrivateKey, btcecPublicKey := btcec.PrivKeyFromBytes(
-		DefaultCurve, operatorPrivateKey.D.Bytes(),
+	btcecPrivateKey, btcecPublickKey := btcec.PrivKeyFromBytes(
```

Comment:
Worth dropping a comment that `btcec.PrivKeyFromBytes` uses the `secp256k1` implementation provided by `decred`  underneath.